packages/storage/src/implementation/metadata.ts

```diff
@@ -69,15 +69,15 @@ export function getMappings(): Mappings {
   if (mappings_) {
     return mappings_;
   }
-  const mappings: Array<Mapping<unknown>> = [];
-  mappings.push(new Mapping('bucket'));
-  mappings.push(new Mapping('generation'));
-  mappings.push(new Mapping('metageneration'));
-  mappings.push(new Mapping('name', 'fullPath', true));
+  const mappings: Array<Mapping<string> | Mapping<number>> = [];
```

Comment:
Is this not just `const mappings: Mappings`?

Reply:
Oh right, done.